classes/components/forms/context/PKPSubmissionsNotificationsForm.inc.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @file classes/components/form/context/PKPSubmissionsNotifications.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPSubmissionsNotifications
+ * @ingroup classes_controllers_form
+ *
+ * @brief A preset form for configuring a context's submissions notifications settings.
+ */
+namespace PKP\components\forms\context;
+use \PKP\components\forms\FormComponent;
+use \PKP\components\forms\FieldHTML;
+use \PKP\components\forms\FieldText;
+use \PKP\components\forms\FieldRadioInput;
+
+define('FORM_SUBMISSIONS_NOTIFICATIONS_SETUP', 'submissionsNotifications');
+
+class PKPSubmissionsNotificationsForm extends FormComponent {
+	/** @copydoc FormComponent::$id */
+	public $id = FORM_SUBMISSIONS_NOTIFICATIONS_SETUP;
+
+	/** @copydoc FormComponent::$method */
+	public $method = 'PUT';
+
+	/**
+	 * Constructor
+	 *
+	 * @param $action string URL to submit the form to
+	 * @param $locales array Supported locales
+	 * @param $context Context Journal or Press to change settings for
+	 */
+	public function __construct($action, $locales, $context) {
+		$this->action = $action;
+		$this->locales = $locales;
+
+		$this->buildCopySubmissionAckPrimaryContactField($context);
+
+		$this->addField(new FieldText('copySubmissionAckAddress', [
+			'label' => __('manager.setup.notifications.copySubmissionAckAddress'),
+			'description' => __('manager.setup.notifications.copySubmissionAckAddress.description'),
+			'size' => 'large',
+			'value' => $context->getData('copySubmissionAckAddress'),
+		]));
+	}
+
+	protected function buildCopySubmissionAckPrimaryContactField($context) {
```

Comment:
Provide a docblock with a description of this method or just move the code directly into the `__construct` method.